src/FSLibrary/StellarStatefulSets.fs

```diff
@@ -60,14 +60,14 @@ type StellarFormation with
             let reinstall = System.Action(installHandler)
 
             if not event.IsSet then
-                self.sleepUntilNextRateLimitedApiCallTime ()
-
-                self.Kube.WatchNamespacedStatefulSetAsync(
-                    name = name,
-                    ``namespace`` = ns,
-                    onEvent = action,
-                    onClosed = reinstall
-                )
+                let fs = sprintf "metadata.name=%s" name
+
+                self
+                    .Kube
```

Comment:
Looks like `WatchNamespacedStatefulSetAsync` is deprecated in 7.0.5, so had to rewrite this a bit.